File: src/test/java/io/github/binout/soccer/domain/date/FriendlyMatchDateTest.java
package io.github.binout.soccer.domain.date;

import org.junit.Test;

import java.time.LocalDate;

import static org.assertj.core.api.Assertions.assertThat;

/**
 * LECTRA
 *
 * @author b.prioux
 */
public class FriendlyMatchDateTest {

    @Test
    public void date_has_no_present_by_default() {
        assertThat(new FriendlyMatchDate(LocalDate.now()).nbPresents()).isZero();
    }

}

File: src/test/kotlin/io/github/binout/soccer/domain/date/FriendlyMatchDateTest.kt
package io.github.binout.soccer.domain.date

import java.time.LocalDate

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class FriendlyMatchDateTest {

    @Test
    fun date_has_no_present_by_default() {
        assertThat(FriendlyMatchDate(LocalDate.now()).nbPresents()).isZero()
    }

}